cookbooks/travis_docker/attributes/default.rb

```diff
@@ -21,3 +20,6 @@
   docker-runc
   dockerd
 ]
+
+# For IBM power
+default['travis_docker']['ppc64le']['apt']['url'] = 'http://ftp.unicamp.br/pub/ppc64el/ubuntu/16_04/docker-17.06.1-ce-ppc64el/'
```

Comment:
As of 17.09, this could possibly use official packages (https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/ppc64el/), or even official static binaries (https://download.docker.com/linux/static/stable/ppc64le/). 😇

Reply:
(Although there's no `trusty` support for `ppc64el` Docker :disappointed:)

Reply:
(But then again, this IBM repo doesn't have 17.09 either :scream:)

Reply:
Yep, I think that this attribute can be dropped, and (later) we can experiment with removing the ppc64le-specific override.